domain_config/src/DomainConfigOverrider.php

```diff
@@ -94,6 +94,14 @@ public function loadOverrides($names) {
         elseif ($override = $this->storage->read($config_name['domain'])) {
           $overrides[$name] = $override;
         }
+
+        // Apply any settings.php overrides.
+        if (isset($GLOBALS['config'][$config_name['langcode']])) {
```

Comment:
`getDomainConfigName()` always returns `'langcode' => 'domain.config.' . $domain->id() . '.' . $this->language->getId() . '.' . $name,`, so $config_name['langcode'] will have a value